typesafe/src/test/scala/zio/config/typesafe/ListSpec.scala

```diff
@@ -0,0 +1,405 @@
+package zio.config.typesafe
+
+import com.typesafe.config.ConfigRenderOptions
+import zio.config._
+import zio.config.magnolia.DeriveConfigDescriptor.descriptor
+import zio.test.Assertion._
+import zio.test.{ DefaultRunnableSpec, suite, test, _ }
+
+object ListSpec extends DefaultRunnableSpec {
```

Comment:
I think I have already done this in separate test. So you could remove `test("A complex enough case for testing HOCON List config and write to/back of HOCON source")`

Reply:
Have a look at `TypesafeConfigReadWriteTest` . Its covering read-write of complex structures

Reply:
updated